modules/serverless-creating-broker.adoc

```diff
@@ -3,41 +3,144 @@
+$ oc apply -f <FILENAME>
 ----
 
-. Give those objects RBAC permissions:
+. Optional. Create an event source that emits events to the `default` Broker. The following example shows how you can create a basic PingSource.
+.. Copy the following into a YAML file:
 +
+
+[source,yml]
 ----
-$ oc -n default create rolebinding eventing-broker-ingress \
-  --clusterrole=eventing-broker-ingress \
-  --serviceaccount=default:eventing-broker-ingress
-$ oc -n default create rolebinding eventing-broker-filter \
-  --clusterrole=eventing-broker-filter \
-  --serviceaccount=default:eventing-broker-filter
+apiVersion: sources.knative.dev/v1alpha2
+kind: PingSource
+metadata:
+  name: ping-source
+spec:
+  schedule: "*/1 * * * *"
+  jsonData: '{"message": "Hello serverless!"}'
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1beta1
+      kind: Broker
+      name: default
 ----
 
-. Create the broker:
 +
+..  Create the event source by applying the YAML file:
+
++
+[source,terminal]
+----
+$ oc apply -f <FILENAME>
+----
+
+. Create the Broker. You can do this by creating a Trigger object that uses the `knative-eventing-injection: enabled` annotation.
++
+[IMPORTANT]
+====
+If you create a Broker using the `knative-eventing-injection: enabled` annotation, you cannot delete this Broker without cluster administrator permissions.
+If you delete the Broker without having a cluster administrator remove this annotation first, the Broker will be created again after deletion.
+====
+
+.. Copy the following into a YAML file:
++
+
+[source,yml]
 ----
-cat << EOF | oc apply -f -
 apiVersion: eventing.knative.dev/v1beta1
-kind: Broker
+kind: Trigger
 metadata:
-  namespace: default
-  name: default <1>
-EOF
+  annotations:
+    knative-eventing-injection: enabled
+  name: example-trigger <1>
+spec:
+  broker: default
+  subscriber: <2>
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+----
+
++
+<1> You can specify any valid name for the Trigger.
+<2> The subscriber configuration is used to specify the event sink. In this example, it uses the Service that was created in step 1 of this procedure.
+
++
+..  Create the Broker by applying the YAML file:
++
+
+[source,terminal]
+----
+$ oc apply -f <FILENAME>
+----
+
+.Verification steps
+
+You can verify that events were sent to the event sink by looking at the sink pod's logs.
+
+By default, Knative Services terminate their pods if no traffic is received within a 60 second period.
+The example shown in this guide creates a PingSource that sends a message every 2 minutes, so each message should be observed in a newly created pod.
+
+. Watch for new pods created:
++
+
+[source,terminal]
+----
+$ watch oc get pods
+----
+
+. Cancel watching the pods using Ctrl+C, then look at the logs of the created pod:
++
+
+[source,terminal]
+----
+$ oc logs $(oc get pod -o name | grep event-display) -c user-container
+----
+
++
+The logs should contain lines similar to the following:
```

Comment:
Please be more specific about what we're looking for in the logs. Are we confirming that the source (138) is what we expect or that the data (142) is present or some other thing?

Reply:
cc @cardil can you clarify please?

Reply:
User should verify that the overall structure of message is the same - that would indicate that proper event was received. Also, user should pay closer attention message ([line 146](https://github.com/openshift/openshift-docs/pull/24504/files#diff-e9f221799bebf77f557343821866f47dR146)), it should be `Hello serverless!` - the same as it was set in previous step of creating PingSource. User should notice other attributes, time should be current, and ID random, other attributes should be the same.

I don't know if we should describe that in such detail, as it should be quite obvious to technical guys out there.

Reply:
Yeah I see what you mean. Once you actually look at what's in there it's fairly obvious that it should match IMHO, especially with the `message` etc., really they're just confirming that it looks OK structure-wise.

@kalexand-rh I think we can assume this level of knowledge for Serverless probably.

Reply:
Thank you both! Please replace this line with `.Example output` before the codeblock - since there's no need to clarify what you're looking for, I think the standard output labelling is better here.